squad/api/rest.py

```diff
@@ -602,10 +602,24 @@ class ProjectStatusSerializer(serializers.HyperlinkedModelSerializer):
 
     def to_representation(self, instance):
         ret = super().to_representation(instance)
+        enriched_details = self.context.get('enriched_details', None)
         if instance.regressions is not None:
-            ret['regressions'] = json.dumps(instance.get_regressions())
+            regressions = instance.get_regressions()
+            if enriched_details:
```

Comment:
am not sure this line is needed since any successfully finished testrun must have environments. maybe i should remove this if statement.